R/dataProcessPlots.R

```diff
@@ -129,20 +136,92 @@ dataProcessPlots = function(
     }
   }
   
-  if (type == "PROFILEPLOT") 
-    .plotProfile(processed, summarized, featureName, ylimUp, ylimDown,
-                 x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
-                 dot.size.profile, width, height, which.Protein, originalPlot, 
-                 summaryPlot, remove_uninformative_feature_outlier, address)
-  if (type == "QCPLOT")
-    .plotQC(processed, featureName, ylimUp, ylimDown, x.axis.size, y.axis.size, 
-            text.size, text.angle, legend.size, dot.size.profile, width, height,
-            which.Protein, address)
-  if (type == "CONDITIONPLOT")
-    .plotCondition(processed, summarized, ylimUp, ylimDown, scale, interval,
-                   x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
-                   dot.size.profile, dot.size.condition, width, height,
-                   which.Protein, save_condition_plot_result, address)
+  if(isPlotly & address != FALSE) {
+      print("Plots will be saved as .HTML file as plotly is selected, set isPlotly = FALSE, if 
+            you want to generate PDF using ggplot2")
+  }
+  
+  if (type == "PROFILEPLOT") {
+      plots <- .plotProfile(processed, summarized, featureName, ylimUp, ylimDown,
+                           x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
+                           dot.size.profile, width, height, which.Protein, originalPlot, 
+                           summaryPlot, remove_uninformative_feature_outlier, address, isPlotly)
+      plotly_plots = list()
+      if(isPlotly) {
+          og_plotly_plot = NULL
+          summ_plotly_plot = NULL
+          if(rlang::has_name(plots, "original_plot")) {
```

Comment:
is it different from "original_plot" %in% names(plots)?

Reply:
works thanks!
Commit [simpler list names search](https://github.com/Vitek-Lab/MSstats/pull/107/commits/2365c82b8789c91b71ea1ea64ddbe0acc1cc07c9)